cmd/dmsg-discovery/internal/api/api.go

```diff
@@ -25,6 +25,9 @@ var log = logging.MustGetLogger("dmsg-discovery")
 
 var json = jsoniter.ConfigFastest
 
+// WhitelistPKs store whitelisted pks of network monitor
+var WhitelistPKs = make(map[string]bool)
```

Comment:
This can be in the `API` struct unexported.

Reply:
Same as address resolver, if we add it on API struct, then cannot use in cli.